src/browser/urls.py

```diff
@@ -112,13 +118,25 @@
         name='browser-file_upload'),
     url(r'^import/(\w+)/(\w+)/file/([ -~]+)', 'browser.views.file_import',
         name='browser-file_import'),
+    url(r'^download/(\w+)/(\w+)/file/([ -~]+)/getcsvapi', 'browser.views.file_download_csv_api'),
```

Comment:
Not sure if you're aware, there's already a file download API, https://datahub.csail.mit.edu/api-docs/#!/repos/File_GET.